config/prompts_memory.py

```
@@ -533,15 +533,93 @@ def get_history_review_prompt(lang: str = 'zh') -> str:
+Rules:
+1. Choose the strongest main emotion, not the safest one.
+2. Do not return neutral unless the text is truly emotionally weak or flat.
+3. Cute, cheerful, playful, affectionate, excited reactions should prefer happy.
+4. Complaining, snapping, irritated, explosive reactions should prefer angry over sad.
+5. Obvious exclamations or unexpected reactions should prefer surprised.
+6. Use sad only when the text clearly shows hurt, disappointment, regret, or low mood.
+7. confidence must be a number between 0 and 1.
+
+Return JSON only, with no explanation.""",
+
+    'ja': """あなたは感情分析の専門家です。入力文の中で最も支配的で外に出ている感情を1つだけ選び、JSONのみで返してください：{"emotion": "emotion_type", "confidence": confidence}。
```

Ruff (0.15.7)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)